### PR TITLE
test: airdrop to deleted accounts

### LINKED ISSUE #14640: (AIRDROP - 60) Given sender attempts to airdrop a token when the receiver account is а deleted ЕОА then the tokenAirdropTransaction should fail. - deleted ED25519 account - deleted ECDSA account

### PR DESCRIPTION
**Pull request by vtronkov**
**Description**:
Adding tests airdropping to different types of deleted accounts

**Related issue(s)**:

Fixes #14640

### CODE REVIEW

File: hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip904/TokenAirdropTest.java [hunk 1]
```diff
@@ -894,7 +958,9 @@ private static SpecOperation[] setUpTokensAndAllReceivers() {
                                 ByteString.copyFromUtf8("c"),
                                 ByteString.copyFromUtf8("d"),
                                 ByteString.copyFromUtf8("e"),
-                                ByteString.copyFromUtf8("f"))),
+                                ByteString.copyFromUtf8("f"),
```
Comment (david-bakin-sl) [inline hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip904/TokenAirdropTest.java:961]: Getting to the point where you could almost consider:

```java
mintToken(NON_FUNGIBLE_TOKEN, 
          "abcdefgh".codePoints()
                    .mapToObj(c->String.valueOf((char)c))
                    .map(ByteString::copyFromUtf8)
                    .toList());
```
Eh, never mind.  Now I look at it isn't clearer.

**Review by david-bakin-sl — APPROVED** [review]
LGTM

File: hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip904/TokenAirdropTest.java (hunk 1 above)
Comment (stoyanov-st) [reply hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip904/TokenAirdropTest.java:961]: Can I suggest this if the list goes on?
```suggestion
                              IntStream.rangeClosed('a', 'h')
                .mapToObj(i -> ByteString.copyFromUtf8(String.valueOf((char) i)))
                .toList();
```

Comment (vtronkov) [reply hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip904/TokenAirdropTest.java:961]: Those are good suggestions, thank you guys! We were thinking about creating a method that will mint a token keep an internal counter and return the serial number as a result. That way you can just call this method and don't worry about the serial number or the mint. We will think about it and we might implement it in a refactoring story and make it more generic for everyone to use.

**Pull request merged**